app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
> Don't you think could also be hard and error prone to double-tap a small capture box though? What if we introduced a artificial delay (500-1000ms) before actually triggering the OCR

I had no issues double tapping a small box, I think introducing a delay does the same thing though, I could double tap, then move the box, then 500ms later, the box is in a different spot.

> It looks like Android will recognize a quick drag + tap again as a onDoubleTap() event.

How in the world do you click that fast? I have been trying to duplicate this for like 10 minutes and have not even once seen it.